src/lib/loaders/loaders_without_authentication/convection.ts

```diff
@@ -0,0 +1,35 @@
+import config from "config"
+import fetch from "node-fetch"
+import urljoin from "url-join"
+
+const { GRAVITY_XAPP_TOKEN, CONVECTION_API_BASE } = config
+
+export const convectionLoaders = (_opts) => {
```

Comment:
It might make more sense to set this up more akin to the way the other services are configured: https://github.com/artsy/metaphysics/blob/f002e67585361a6ce3558c955772c9a66b99524a/src/lib/loaders/loaders_without_authentication/diffusion.ts#L1 for instance, and be able to reuse the convection fetch: https://github.com/artsy/metaphysics/blob/f002e67585361a6ce3558c955772c9a66b99524a/src/lib/apis/convection.ts

Reply:
Another reason(s) to thread this through using that setup a la the _with_ authentication one: https://github.com/artsy/metaphysics/blob/f002e67585361a6ce3558c955772c9a66b99524a/src/lib/loaders/api/index.ts#L166-L173 , is that it threads through the originating user agent of the request, plus request id (can be used to associated a single user-initiated request with API calls in logs).

Reply:
Ahhh. This makes a lot of sense. My approach bypasses this and we lose logging, caching for GET requests etc. 
Thanks @mzikherman . Will refactor.